### ISSUE TITLE
Usage of `Type.name` will always throw in v0.6

### ISSUE THREAD

**Issue by m-j-w** [KristofferC/OhMyREPL.jl#57]
In https://github.com/KristofferC/OhMyREPL.jl/blob/master/src/errormessage_overrides.jl#L146, I believe `Type.name` to be incorrect when used in Julia v0.6. The error is `Type UnionAll has no field name`.
This error only occured indirectly when I accidentally used an older commit prior to the fixes for v0.6.

**Comment by KristofferC** [comment]
Since 0.6 is a moving target, I think that 0.6 support should only be expected to work when using the latest version of OhMyREPL. Do you agree?

**Comment by m-j-w** [comment]
Sure. I simply followed your advice in the issue linked above. The main point is that the comparison `ft.name === Type.name` is still in the code and will always throw if ever invoked, since `Type.name` does not exist anymore in v0.6.
```
julia> Type.name
ERROR: type UnionAll has no field name

julia> Int64.name === Type.name
ERROR: type UnionAll has no field name
```
Its probably not an issue requiring immediate action.

**Comment by KristofferC** [comment]
Yes, however, this code is currently only included on 0.5.
